packages/commonwealth/client/scripts/views/components/quill/quill_editor.ts

```diff
@@ -112,7 +112,12 @@ export class QuillEditor extends QuillEditorInternal {
 
   public isBlank(): boolean {
     if (this._quill.editor.isBlank()) return true;
-    if (this._quill.getText() === '\n') return true;
+    const newLinesOnly = /^\s+$/.test(this.text);
+    if (this.markdownMode) {
+      if (newLinesOnly) return true;
```

Comment:
Technically we can just return the truth values rather than if/true (e.g. `return newLinesOnly` and `return this.contents.ops.length === 1 && newLinesOnly`).